src/amp/components/Epic.tsx

```diff
@@ -0,0 +1,168 @@
+    background-color: ${palette.neutral[97]};
+    clear: left;
+    margin-top: 1.5rem;
+    margin-bottom: 1.5rem;
+    padding: 0.25rem 0.3125rem 1rem;
+`;
+const epicHeader = css `
+    font-size: 1.25rem;
+    line-height: 1.4375rem;
+    ${headline.xxsmall()};
+    text-rendering: optimizeLegibility;
+    font-kerning: normal;
+    font-variant-ligatures: common-ligatures;
+    font-weight: 900;
+    margin-bottom: 0.75rem;
+    -webkit-font-smoothing: antialiased;
+`;
+const epicParagraph = css`
+    font-size: 1.1rem;
+    display: block;
+    margin-block-start: 0.5rem;
+    margin-block-end: 0.5rem;
+
+    ${body.medium()};
+    text-rendering: optimizeLegibility;
+    font-kerning: normal;
+    font-variant-ligatures: common-ligatures;
+    -webkit-font-smoothing: antialiased;
+    vertical-align: 0%;
+    line-height: 1.5;
+    &::selection {
+        background-color: ${palette.brandYellow.main};
+    }
+    &:last-of-type {
+      display: inline;
+    }
+`;
+const highlightedText = css`
+    font-size: 1.1rem;
+    background-color: ${palette.brandYellow.main};
+    padding: 0.125rem;
+    margin-left: 5px;
+    color: ${palette.neutral[7]};
+    ${headline.xxxsmall({fontWeight: "bold"})};
+    text-rendering: optimizeLegibility;
+    font-kerning: normal;
+    font-variant-ligatures: common-ligatures;
+    -webkit-font-smoothing: antialiased;
+    vertical-align: 0%;
+    line-height: 1.5;
+    display: inline;
+`;
+const supportButton = css`
+    background-color: ${palette.brandYellow.main};
+    color: ${palette.neutral[7]};
+    display: inline-block;
+    ${textSans.medium()};
+    text-rendering: optimizeLegibility;
+    font-kerning: normal;
+    font-variant-ligatures: common-ligatures;
+    -webkit-font-smoothing: antialiased;
+    align-items: center;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 1.1rem;
+    height: 2.625rem;
+    min-height: 2.625rem;
+    padding: 0 1.3125rem;
+    border: 0;
+    border-radius: 1.3125rem;
+    box-sizing: border-box;
+    cursor: pointer;
+    margin: 2rem 0.625rem 0.25rem 0;
+    vertical-align: base;
+    line-height: 2.625rem;
+    transition: background-color .3s;
+    text-align: centre;
+    &:hover {
+        background-color: ${palette.opinion[600]};
+    }
+`;
+const arrow = css`
+    margin-left: .5rem;
+    position: relative;
+    width: 1.3125rem;
+    height: auto;
+    display: inline;
+    color: ${palette.neutral[7]};
+    vertical-align: sub;
+`;
+const acceptedPaymentMethodsWrapper = css`
+    margin-top: .5rem;
+    margin-left: .5rem;
+    display: block;
+`;
+
+export const Epic: React.FC<{
+    pageId: string
+}> = ({pageId}) => {
+    const testArticleId = 'science/2015/sep/28/nasa-scientists-find-evidence-flowing-water-mars'
+    const epicUrl = process.env.NODE_ENV === 'production' ?
+        'https://contributions.guardianapis.com/amp/epic' :
+        'https://contributions.code.dev-guardianapis.com/amp/epic';
```

Comment:
Could/should these URLs be in config somewhere?

Reply:
I think that's one for @tomrf1 to look at tomorrow, he added those and may have had a reason for doing it this way that I'm unaware of.

Reply:
I'm not sure there's much benefit to wiring it up with config.
Checking `NODE_ENV` seems to be quite common in this repo